cardano-tracer/src/Cardano/Tracer/Acceptors/Server.hs

```diff
@@ -64,36 +60,19 @@ runAcceptorsServer
   -> IO ()
 runAcceptorsServer tracerEnv tracerEnvRTView howToConnect ( ekgConfig, tfConfig, dpfConfig) =
   withIOManager \iocp -> do
-  traceWith (teTracer tracerEnv) $ TracerSockListen (Net.howToConnectString howToConnect)
-  case howToConnect of
```

Comment:
Did you accidentally remove scrutinizing `howToConnect` (and branching correctly between `doListenToForwarderSocket` and `doListenToForwarderLocal`, renaming the latter to `doListenToForwarder`)?